k8s/eks/yaml/multus-daemonset.yml

```diff
@@ -0,0 +1,281 @@
+# Note:
```

Comment:
Explained [here ](https://github.com/testground/testground/issues/1457)why we use locally hosted multus.

Reply:
Let's include information about specific versions/commit hashes that caused this. Let's do this in the issue and ensure that a comment here links to it.

Reply:
This was caused by a complete overhaul in multus logic. In v3.9.1 they started migrating to the client/server model.
This new deployment is called 'thick plugin', in contrast to the previous deployment, which is now called 'thin plugin'.

So once we saw the changes in the documentation, we changed the names of the images our version of multus was pulling and it deployed correctly again.

Reply:
Followed up in https://github.com/testground/testground/issues/1457#issuecomment-1264989166